src/Plugin/Action/IdentifierAction.php

```diff
@@ -125,7 +125,7 @@ public function getIdentifier(): IdentifierInterface {
    * @throws \Drupal\Core\Entity\Exception\UndefinedLinkTemplateException
    */
   public function getExternalUrl(): string {
-    return $this->entity->toUrl()->setAbsolute()->toString(TRUE)->getGeneratedUrl();
+    return $this->entity->toUrl()->setAbsolute()->setOption('alias', TRUE)->toString(TRUE)->getGeneratedUrl();
```

Comment:
Unsure if there might be a point at which we might want to move the misc options being set into the array, as in:

```suggestion
    return $this->entity->toUrl(options: [
      'absolute' => TRUE,
      'alias' => TRUE,
    ])->toString(TRUE)->getGeneratedUrl();
```

... but yeah, probably fine as-is, for now. If we find ourselves jamming in more options, we can address then.